infrahub_sdk/template/__init__.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import annotations
+
+import linecache
+from pathlib import Path
+from typing import Any, Callable, NoReturn
+
+import jinja2
+from jinja2 import meta
+from jinja2.sandbox import SandboxedEnvironment
+from netutils.utils import jinja2_convenience_function
+from rich.syntax import Syntax
+from rich.traceback import Traceback
+
+from .constants import CURRATED_NETUTILS_FILTERS
+from .exceptions import (
+    JinjaTemplateError,
+    JinjaTemplateNotFoundError,
+    JinjaTemplateOperationViolationError,
+    JinjaTemplateSyntaxError,
+    JinjaTemplateUndefinedError,
+)
+from .models import UndefinedJinja2Error
+
+netutils_filters = jinja2_convenience_function()
```

Comment:
curious why we have this function at the root of the file, if possible I think it would make sense to move it inside Jinja2Template

Reply:
Just a slight performance thing since the function would check to see if the modules have been imported each time we render a template. But I can move it into the class instead, we'd have to have a pretty significant number of templates rendered for it to be an issue.

Reply:
all good I was just curious